[32m2023-12-18 14:25:58 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 07958382-4d98-4837-9e93-8cc631ab095c - 7715 - LOGS_CAPTURED - Started capturing logs in process (pid: 7715).
[32m2023-12-18 14:25:58 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 07958382-4d98-4837-9e93-8cc631ab095c - 7715 - PercentageFullTimeAndPartTimePerDepartment - STEP_START - Started execution of step "PercentageFullTimeAndPartTimePerDepartment".
[32m2023-12-18 14:25:58 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 07958382-4d98-4837-9e93-8cc631ab095c - PercentageFullTimeAndPartTimePerDepartment - Loading file from: /Users/nirmalgeorge/dap_dagster_project/DAP-project/govt_employee_analysis/tmpjy9nrer1/storage/LoadingCsv using PickledObjectFilesystemIOManager...
[32m2023-12-18 14:25:58 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 07958382-4d98-4837-9e93-8cc631ab095c - 7715 - PercentageFullTimeAndPartTimePerDepartment - ASSET_OBSERVATION - DagsterEventType.ASSET_OBSERVATION for step PercentageFullTimeAndPartTimePerDepartment
[32m2023-12-18 14:25:58 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 07958382-4d98-4837-9e93-8cc631ab095c - 7715 - PercentageFullTimeAndPartTimePerDepartment - LOADED_INPUT - Loaded input "LoadingCsv" using input manager "io_manager"
[32m2023-12-18 14:25:58 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 07958382-4d98-4837-9e93-8cc631ab095c - 7715 - PercentageFullTimeAndPartTimePerDepartment - STEP_INPUT - Got input "LoadingCsv" of type "Any". (Type check passed).
[32m2023-12-18 14:25:58 +0000[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - 07958382-4d98-4837-9e93-8cc631ab095c - 7715 - PercentageFullTimeAndPartTimePerDepartment - STEP_FAILURE - Execution of step "PercentageFullTimeAndPartTimePerDepartment" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "PercentageFullTimeAndPartTimePerDepartment"::

ValueError: Could not interpret input 'full_or_part_time'

Stack Trace:
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 465, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
                                                                    ^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/dap_dagster_project/DAP-project/govt_employee_analysis/govt_employee_analysis/assets.py", line 119, in PercentageFullTimeAndPartTimePerDepartment
    sns.barplot(data = proportions_reset, x = 'department', y = 'full_or_part_time')
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/seaborn/categorical.py", line 2755, in barplot
    plotter = _BarPlotter(x, y, hue, data, order, hue_order,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/seaborn/categorical.py", line 1530, in __init__
    self.establish_variables(x, y, hue, data, orient,
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/seaborn/categorical.py", line 541, in establish_variables
    raise ValueError(err)
[0m
